### ISSUE TITLE
Use different strings for `aria-label` attributes of room sublist and its header

### ISSUE THREAD

**Issue by luixxiul** [vector-im/element-web#25118]
### Steps to reproduce

1. Create a couple of rooms
2. Check ARIA label attributes of `Rooms` list (`mx_RoomSublist`) and its header (`mx_RoomSublist_headerContainer`)

#### What did you expect?

Those attributes should be different like below.

![2](https://user-images.githubusercontent.com/3362943/232208674-d6e79d81-2635-482a-8b02-c6b8d834c785.png)

#### What happened instead?

They are same.

![1](https://user-images.githubusercontent.com/3362943/232208671-f4a3ee6c-eb5d-4b17-b18e-985723dd69df.png)

### URL for webapp

localhost

### Application version

develop branch

### Will you send logs?

No

**Comment by luixxiul** [comment]
Now I am not sure if using different strings is sensible.
